pr_consistency/1.get_merged_prs.py

```diff
@@ -88,21 +96,24 @@ def parse_isoformat(string):
             continue
 
         if str(pr.number) in pull_requests:
-            if pr.updated_at > parse_isoformat(pull_requests[str(pr.number)]['updated']):
-                print("Updating entry for pull request #{} ({} of max {})".format(pr.number, i, max_issues))
+            if (pr.updated_at.replace(tzinfo=None) >
```

Comment:
Note: The `pr` datetime stuff is now timezone-aware, which causes error when compared to timezone-unaware data from JSON file. I found this workaround from https://stackoverflow.com/questions/796008/cant-subtract-offset-naive-and-offset-aware-datetimes .

Content from JSON **before** update:
```json
  "6956": {
    "created": "2017-12-08T23:34:08",
    "labels": [
      "io.fits"
    ],
    "merge_commit": "6dbd31afc91065a67e96c1b3dd880ac2592b022b",
    "merged": "2018-01-24T19:21:53",
    "milestone": "v2.0.4",
    "title": "Fix behavior of scaled columns in binary table",
    "updated": "2018-01-24T19:37:47"
  },
```

Content to JSON **after** update:
```json
  "6956": {
    "created": "2017-12-08T23:34:08+00:00",
    "labels": [
      "io.fits"
    ],
    "merge_commit": "6dbd31afc91065a67e96c1b3dd880ac2592b022b",
    "merged": "2018-01-24T19:21:53+00:00",
    "milestone": "v2.0.4",
    "title": "Fix behavior of scaled columns in binary table",
    "updated": "2018-01-25T06:37:35+00:00"
  },
```